docs/AboutUs.md

```diff
@@ -52,8 +57,9 @@ We are a team based in the [School of Computing, National University of Singapor
 
 <img src="images/giang.png" width="200px">
```

Comment:
Should be pcgiang.png

Reply:
Update the name of Giang's image inside the images folder too.

Reply:
Updated changes

Reply:
No, **do not change the github account link**!